engines/group/app/serializers/group/base_serializer.rb

```diff
@@ -0,0 +1,5 @@
+module Group
+  class BaseSerializer < Oat::Serializer
+    adapter FlatAPI
```

Comment:
The place for this should be the `Shared` engine actually, but since we would include a module that only has one line I left it as it is.